android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/SocialSession.kt

```diff
@@ -0,0 +1,13 @@
+package io.logto.sdk.android.auth.social
+
+import android.app.Activity
+import io.logto.sdk.android.completion.Completion
+
+interface SocialSession {
+    val context: Activity
+    val redirectTo: String
```

Comment:
discussion: should we create session with `redirectTo` and `callbackUri` or like swift SDK, pass them in `.start()`

cc @logto-io/eng